core_backend/app/routers/question_answer.py

```diff
@@ -87,9 +89,17 @@ async def get_llm_answer(
             user_query_refined, qdrant_client, int(QDRANT_N_TOP_SIMILAR)
         )
         response.content_response = content_response
-        response.llm_response = await get_llm_rag_answer(
+        llm_response = await get_llm_rag_answer(
             user_query_refined.query_text, content_response[0].retrieved_text
         )
+
+        if llm_response == ANSWER_QUESTION_ERROR_MESSAGE:
+            error_response = UserQueryResponseError(
+                query_id=response.query_id,
+                error_message=ANSWER_QUESTION_ERROR_MESSAGE,
+                error_type=ErrorType.LLM_RESPONSE_FAILED,
+            )
+            return error_response
```

Comment:
The reason we don't want to do this is the same as the discussion [here](https://github.com/IDinsight/aaq-core/pull/87/files#r1466206069). Even if summarisation fails, we still want to allow the user chat manager to browse the FAQs (the flow for this will be set up on the chat manager side).